cg/constants/demultiplexing.py

```diff
@@ -18,6 +19,15 @@ class FlowCellType(StrEnum):
     HISEQ: str = "hiseq"
 
 
+class SequencingInstruments(StrEnum):
+    """Define properties of the sequencing instruments."""
+
+    NOVASEQXPLUS: str = "NovaSeqXPlus"
```

Comment:
Do we need the PLUS part everywhere? I assume the format is identical to `NovaSeqX`? If not disregard this

Reply:
I can double-check with @henrikstranneheim

Reply:
I do not know maybe @karlnyr knows. If they are let us skip the plus